techlibs/quicklogic/qlf_k6n10f/generate_bram_types_sim.py

```diff
@@ -1,10 +1,17 @@
+import os
 import sys
+import time
 from datetime import datetime, timezone
 
 def generate(filename):
+	build_date = datetime.fromtimestamp(
+		int(os.environ.get('SOURCE_DATE_EPOCH', time.time())),
+		tz=timezone.utc,
+	)
+
 	with open(filename, "w") as f:
 		f.write("// **AUTOGENERATED FILE** **DO NOT EDIT**\n")
-		f.write(f"// Generated by {sys.argv[0]} at {datetime.now(timezone.utc)}\n")
+		f.write(f"// Generated by {sys.argv[0]} at {build_date}\n")
```

Comment:
Do we even need the build date here? The file modification time indicates it already...

Reply:
I don't think it's *particularly* useful, but meh, I don't care either way, as long as the file doesn't change between rebuilds ;)

Reply:
I think you can go ahead and remove the generation time outright. It's an outlier among the other "Generated by" string formatting in the codebase

Reply:
Updated.